app/Http/Requests/NewPasswordRequest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Support\Facades\DB;
+
+class NewPasswordRequest extends FormRequest
+{
+
+    /**
+     * Indicates if the validator should stop on the first rule failure.
+     *
+     * @var bool
+     */
+    protected $stopOnFirstFailure = true;
+
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true; // 'false' - for only registered users, 'true' - for all users
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, mixed>
+     */
+    public function rules()
+    {
+
+        return [
+            "token" => [
+                "exists:reset_passwords,token",
+                function ($attribute, $value, $fail) {
```

Comment:
create a separate validation rule and use it here
https://laravel.com/docs/9.x/validation#custom-validation-rules